File: src/main/java/org/tech/blog/reactivedemo/controller/Ready.java
package org.tech.blog.reactivedemo.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class Ready {
    @GetMapping("/hello")
    public String hello() {
        return "Hello, world";
    }
}

File: src/main/kotlin/org/tech/blog/reactivedemo/controller/Ready.kt
package org.tech.blog.reactivedemo.controller

import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class Ready {
    @GetMapping("/hello")
    fun hello(): ResponseEntity<String> {
        return ResponseEntity.ok("hello world")
    }
}